test/conformance/conformance_arena_constraints.cpp

```diff
@@ -79,12 +79,17 @@ TEST_CASE("Test core types topology traversal correctness") {
 TEST_CASE("Test validity of NUMA interfaces when HWLOC is not present on the system") {
     std::vector<oneapi::tbb::numa_node_id> numa_indexes = oneapi::tbb::info::numa_nodes();
 
+#if __TBB_PORTABLE_TBBBIND_AVAILABLE
+    REQUIRE_MESSAGE(numa_indexes[0] != -1,
```

Comment:
I am not sure why `-1` is used below, but is not that there is `automatic` value for this or are these values unrelated?

Reply:
Constraints are set to `task_arena::automatic` by default. Don't know why in wasn't used in this test.

Reply:
Spec https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onetbb/source/info_namespace#_CPPv410numa_nodesv

Reply:
Changed.